organization/_posts/2017-11-15-packaging.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "HSF Packaging Group Meeting #14, November 15, 2017"
+layout: default
+---
+
+# HSF Packaging Group Meeting #14, November 15, 2017
+
+#### *Present*: Ben Morgan, Chris Green, Dario Menasce, Graeme Stewart, Patrick Gartung, Martin Ritter, Emil Obreshkov, Javier, Rafal, Guilherme
```

Comment:
Oh, I meant to fill in these names properly (I am bad with spelling surnames). Should be

Javier Cervantes Villanueva, Raphal Pacholek, Guilherme Amadio